SpatialGDK/Documentation/content/dynamic-typebindings.md

```diff
@@ -0,0 +1,31 @@
+# Dynamic Typebindings
+To allow Unreal to replicate through the SpatialOS network stack and combine multiple dedicated server instances across one seamless game world, you need to adapt Unreal networking functionality to work with the [SpatialOS Worker API (SpatialOS documentation)](https://docs.improbable.io/reference/latest/capi/introduction). You can do this seamlessly using `Dynamic Typebindings`. `Dynamic Typebindings` operate at runtime so that your iteration speed is not affected despite your network code running on a completely different representation than Unreal’s.
```

Comment:
Should we say "we needed to adapt"? Because this sounds like this is required by the user.